src/store/middleware/boardReaction.tsx

```diff
@@ -0,0 +1,10 @@
+import {Dispatch, MiddlewareAPI} from "@reduxjs/toolkit";
+import {ApplicationState} from "types";
+import {Action, ReduxAction} from "store/action";
+import {API} from "api";
+
+export const passBoardReactionMiddleware = (stateAPI: MiddlewareAPI<Dispatch, ApplicationState>, dispatch: Dispatch, action: ReduxAction) => {
+  if (action.type === Action.AddBoardReaction) {
+    API.addBoardReaction(action.context.board!, action.reactionType); // TODO handle promise & toasts
```

Comment:
is this TODO still to do? :)

Reply:
depends on whether we want to do something in case something goes wrong. do we want to display a toast or is that too much for something as small as a board reaction?

Reply:
I'd say it's too much